packages/react-textarea/MIGRATION.md

```diff
@@ -0,0 +1,36 @@
+# TextArea Migration
+
+## STATUS: WIP
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0
+
+- `TextArea`
```

Comment:
do we need to have all of the props? I was only showing the props that changed

Reply:
yeah, otherwise i have to parse the whole list to see which ones have changed.

Reply:
Sounds good, I'll make the changes!